Makefile

```diff
@@ -11,10 +11,11 @@ DIFF = diff -q
 test: test/build \
 	test-help \
 	test-version \
-	$(patsubst %,test/build/%.css,$(TESTS)) \
 	test-multi \
+	test-context \
 	test-replace \
 	test-local-plugins
+	# $(patsubst %,test/build/%.css,$(TESTS)) \
```

Comment:
I had commented the line out because it was failing (and failing fast), therefore not reaching my new test task. This shouldn't have made it into the commit. I'll un-comment the line and update my pull request. Thanks for catching it!